flow_py_sdk/account_key.py

```diff
@@ -54,3 +88,54 @@ def from_proto(cls, k: entities.AccountKey) -> AccountKey:
         ak.sequence_number = k.sequence_number
 
         return ak
+
+    @classmethod
+    def from_seed(cls, sign_algo: SignAlgo, hash_algo: HashAlgo, seed: str = None ) -> tuple[AccountKey, in_memory_signer.InMemorySigner]:
+        """
+        from_seed provide a way for user to create a public and private key for an account using, seed string.
+
+        Parameters
+        ----------
+        sign_algo : int
+            Signature algorithm associate with account.
+        hash_algo : str
+            Hash algorithm associate with account.
+        seed : str
+            The seed associate with account.
+
+        Returns
+        -------
+        َAccountKey
```

Comment:
A weird character snuk in here.
```suggestion
        AccountKey
```